### PR TITLE
two typos and a suggestion for an addition

### PR DESCRIPTION
**Pull request by hubwoop**

### CODE REVIEW

File: 06ln-reflection-annotations.md [hunk 1]
```diff
@@ -265,7 +265,7 @@ However, newer versions of Java have a sophisticated [system of permissions and
 Two things that do _not_ work, at least out of the box:
 - While you can do a forced write on `final` _fields_, this typically does not affect the code at runtime, since the values are already bound at compiletime.
 - It is impossible to swap out _methods_ since class definitions are read-only and read-once.
-	If you wanted to facilitate that, you would have to write your own class loader.
+	If you wanted to facilitate that, you would have to write your own class loader or you could use the decorator pattern.
```
Comment (sikoried) [inline 06ln-reflection-annotations.md]: The decorator pattern wouldn't help here, since the original method would still be in place (via `super`).

**Review by sikoried — CHANGES_REQUESTED** [review]
I don't agree on the decorator one, but the rest looks good to me!

File: 06ln-reflection-annotations.md (hunk 1 above)
Comment (hubwoop) [reply 06ln-reflection-annotations.md]: understood, thanks.

**Review by sikoried — APPROVED** [review]

**Comment by sikoried** [comment]
Thanks!

**Pull request merged**